subprojects/docs/src/samples/userguide/initScripts/customLoggerKts/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+tasks.create("compile") {
+    doLast {
+        println("compiling source")
+    }
+}
+tasks.create("testCompile") {
+    dependsOn("compile")
+    doLast {
+        println("compiling test source")
+    }
+}
+tasks.create("test") {
+    dependsOn("compile", "testCompile")
+    doLast {
+        println("running unit tests")
+    }
+}
+tasks.create("build") {
+    dependsOn("test")
+}
```

Comment:
Do you want to use the new deferred task configuration API?

Reply:
The side by side Groovy and Kotlin samples should behave the same way. Moving to the configuration avoidance APIs should be done separately and uniformly.